### PR TITLE
docs: correct docs for `LoadJobConfig.destination_table_description`

### LINKED ISSUE #811: Correct docstring for LoadJobConfig.destination_table_description property
The docstring refers to the "name" of the table, which is incorrect.

### PR DESCRIPTION
**Pull request by mgorsk1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #811.

### CODE REVIEW

**Comment by mgorsk1** [comment]
@googlebot I signed it!

**Comment by mgorsk1** [comment]
@googlebot I fixed it.

File: google/cloud/bigquery/job/load.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def destination_encryption_configuration(self, value):
 
     @property
     def destination_table_description(self):
-        """Optional[str]: Name given to destination table.
+        """Optional[str]: Description given to destination table.
```
Comment (plamut) [inline google/cloud/bigquery/job/load.py:173]: I'm not a native speaker, but this doesn't sound right? We don't really "give" description to an object.

```suggestion
        """Optional[str]: Description of the destination table.
```

**Review by plamut — CHANGES_REQUESTED** [review]
Thanks for spotting this and taking the initiative to fix it yourself! FWIW, I created an issue and linked it with this PR.

I believe the sentence should be re-worded, after which we can merge this.

File: google/cloud/bigquery/job/load.py (hunk 1 above)
Comment (mgorsk1) [reply google/cloud/bigquery/job/load.py]: you're right, fixed this with last commit

**Review by plamut — APPROVED** [review]
Looks good, thanks again for your contribution!

**Comment by mgorsk1** [comment]
My pleasure! Thanks for prompt review.

**Comment by plamut** [comment]
Changed the PR title to make the commit message bot happy, no need to update the PR itself.

**Pull request merged**